net-im/bitlbee-facebook/bitlbee-facebook-9999.ebuild

```diff
@@ -25,6 +25,7 @@ RDEPEND="
 	dev-libs/json-glib
 	>=net-im/bitlbee-3[plugins]"
 DEPEND="${RDEPEND}
```

Comment:
EAPI-7 bump requires changes in this area. keep the `DEPEND` line, but close it after "${RDEPEND}".
Both `dev-util/glib-utils` and `virtual/pkgconfig` need to go into `BDEPEND`.

Reply:
I did this change in its own commit.